mapbox_mcp/index.ts

```
@@ -1,64 +1,92 @@
 
   try {
-    console.log(`Attempting to connect to Composio Mapbox...`);
+    // Log the connection attempt (masking API key for security).
+    const urlToLog = serverUrl.split('?')[0] + `?profile=${profileId}&api_key=****`;
+    console.log(`Attempting to connect to MCP server at ${urlToLog}...`);
     
-    // Initialize the Composio client and authenticate
-    const { connectionId, connectedAccount } = await initializeComposioMapbox();
-    composioClient = getComposioClient();
+    // Initialize the MCP client using createMcpClient (assumed Node.js equivalent of useMcp).
+    client = await useMcp({
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)